test/e2e/serviceloadbalancers.go

```diff
@@ -245,8 +245,13 @@ var _ = Describe("ServiceLoadBalancer", func() {
 })
 
 // simpleGET executes a get on the given url, returns error if non-200 returned.
-func simpleGET(c *http.Client, url string) (string, error) {
-	res, err := c.Get(url)
+func simpleGET(c *http.Client, url, host string) (string, error) {
```

Comment:
Same comment as above.  Can we move this out of this test to a utility library.
Happy to defer to a separate PR if you prefer.  Just create a github issue and reference it here.